src/test/scala/org/broadinstitute/dsde/workbench/sam/RetryableAnyFlatSpec.scala

```diff
@@ -7,7 +7,7 @@ trait RetryableAnyFlatSpec extends AnyFlatSpec with Retries {
 
   val retries = 3
 
-  override def withFixture(test: NoArgTest): Outcome =
+  override protected def withFixture(test: NoArgTest): Outcome =
```

Comment:
this is `protected` in the parent class, making it public here causes conflicts when mixing in other traits that have a similar definition